### PR TITLE
Update doc-ru-config-values.yaml

### PR DESCRIPTION
**Pull request by MariaMakeeva**
# Description

The Russian-language text has been unified.

## Why do we need it, and what problem does it solve?

It does not affect critical components.

## Why do we need it in the patch release (if we do)?

Need to add a Russian-language text.

## What is the expected result?

The Russian version should be displayed on the site page.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [x] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: istio
type: fix 
summary: Update documentation.
impact_level:  low
```

### CODE REVIEW

File: modules/110-istio/openapi/doc-ru-config-values.yaml [hunk 1]
```diff
@@ -210,51 +210,65 @@ properties:
...
-                      Лимиты для памяти.
+                      Настройка ограничений (limits) использования CPU и памяти.
+            properties:
+              cpu:
+                description: |
+                  Настройка ограничений использования CPU (CPU limits).
+              memory:
+                description: |
+                  Настройка ограничений использования памяти (memory limits).
+
```
Comment (MariaMakeeva) [inline modules/110-istio/openapi/doc-ru-config-values.yaml:271]:
```suggestion
                      Настройка ограничений (limits) использования CPU и памяти.
                properties:
                   cpu:
                       description: |
                           Настройка ограничений использования CPU (CPU limits).
                   memory:
                        description: |
                            Настройка ограничений использования памяти (memory limits).

```

Comment (MariaMakeeva) [reply modules/110-istio/openapi/doc-ru-config-values.yaml:271]: Тут просто иерархию поправила.

File: modules/110-istio/openapi/doc-ru-config-values.yaml [hunk 2]
```diff
@@ -139,24 +139,24 @@ properties:
           mode:
           static:
             description: |
-              Настройки управления ресурсами в статическом режиме.
+              Настройки управления ресурсами в режиме `Static`
```
Comment (MariaMakeeva) [inline modules/110-istio/openapi/doc-ru-config-values.yaml:142]:
```suggestion
              Настройки управления ресурсами в режиме `Static`.
```

File: modules/110-istio/openapi/doc-ru-config-values.yaml [hunk 3]
```diff
@@ -210,51 +210,65 @@ properties:
...
+                    Максимальное значение, которое может выставить VPA для запроса CPU (CPU requests).
                 limitRatio:
                   description: |
-                    Коэффициент для вычисления лимитов для CPU. Опционально используется для подсчета первичных лимитов для CPU у пода. VPA при своей работе сохраняет пропорцию лимиты/реквесты и при вычислении подходящих актуальных реквестов настраивает лимиты в том же соотношении.
-              memory:
+                    Коэффициент расчета ограничений использования CPU относительно запросов (limits/requests).
+                    
+                    Применяется для расчета начальных ограничений использования CPU для пода (CPU limits).
+                    
+                    Если параметр указан, то VPA, при пересчете запросов CPU (CPU requests) будет пересчитывать ограничения использования CPU (CPU limits) сохраняя указанное соотношение ограничений к запросам (limits/requests).              memory:
```
Comment (MariaMakeeva) [inline modules/110-istio/openapi/doc-ru-config-values.yaml:231]:
```suggestion
                    Если параметр указан, то VPA, при пересчете запросов CPU (CPU requests) будет пересчитывать ограничения использования CPU (CPU limits) сохраняя указанное соотношение ограничений к запросам (limits/requests).
                                  memory:
```

Comment (MariaMakeeva) [reply modules/110-istio/openapi/doc-ru-config-values.yaml:231]: memory: убежало. 

Может быть в 233 строку вставить?

**Comment by z9r5** [comment]
Fixed in https://github.com/deckhouse/deckhouse/pull/7421/files

**Pull request closed without merging**